api/src/main/java/com/exelaration/abstractmemery/AbstractMemeryApplication.java

```diff
@@ -2,12 +2,27 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = {SecurityAutoConfiguration.class})
 public class AbstractMemeryApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(AbstractMemeryApplication.class, args);
 	}
 
-}
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurer() {
+			@Override
+			public void addCorsMappings(CorsRegistry registry) {
+				registry.addMapping("/upload").allowedOrigins("http://localhost:3000");
+			}
+		};
+	}
+
+
+}
```

Comment:
This is being done at the Controller level.  Why are we also doing it at an application level?  I prefer the express intent of doing it at the lower level - either Controller or Action.